packages/components/src/components/ui-shell/_content.scss

```diff
@@ -15,7 +15,7 @@
 @mixin carbon-content {
   .#{$prefix}--content {
     padding: 2rem;
-    background: $ui-02;
+    background: $layer;
```

Comment:
This changes the background color for the UI Shell content tag.

Everywhere else, `$ui-01` is being replaced with `$layer`. This is the only place where `$ui-02` is being replaced. Is this an error?

Reply:
This was an intentional change: https://github.com/carbon-design-system/carbon/pull/8376#discussion_r613421974

cc @aagonzales

Reply:
Hi @tw15egan , I run the same code but on my local, content background color is ui-02, but on cloud the color is ui-01. I use a custom theme: (ui-background: #ffffff, ui-01: #f3f3f3, ui-02: #ffffff).
Do you know what make that difference? What should I do to make it run consistently on both environment?

Reply:
Your local my be out of date and not pulling in the most recent changes. Seems like cloud is pulling in the latest that has changed it from `ui-02` to `ui-01` (`layer` in v11)

Reply:
@tw15egan , you're right! I'll pull the new code changes. Thanks!